configuration/config2/configfiles/newenv.xsd

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does this XSD represent the overall environment configuration definition? If so, the filename should reflect that.

Reply:
We discussed. This is for testing the layout of the overall environment.xsd. I'll start building that one soon.